README.md

```diff
@@ -88,7 +79,66 @@ println(response)
 )
 */
 ```
+#### Currently only two backend implementations are available:
+* `OpenAISyncBackend` which uses `Identity` as an effect `F[_]`
```

Comment:
**nitpick:** Although `Id[A]` is called _identity monad_, here it implies that it's the type name, so maybe it should be 
```
which uses identity monad `Id[A]` as an effect `F[A]`
```